subjects/prime_checker/README.md

```diff
@@ -0,0 +1,43 @@
+## prime_checker
+
+### Instructions
+
+Create a **function** `prime_checker` that takes an `u32` and check if it is a prime number.
+
+The result will be `None` if the argument is less or equal one otherwise, it will return the result with the prime number inside if the number is prime, or it will return an `enum` `PrimeErr` if the `u32` is not prime.
```

Comment:
Here maybe let's say "the function will return an `Option`. It will be `None` if... and `Result` otherwise".
Then we could specify the `Result` values.